bids_conversion/convert_somato_data.py

```diff
@@ -0,0 +1,268 @@
+"""Convert the MNE-somato-data to BIDS format.
+
+Requirements
+------------
+The python dependencies to run this script are:
+
+- numpy>=1.16
```

Comment:
why this?

Reply:
because that's the version I used during testing it :-)